src/ArborX_LinearBVH.hpp

```diff
@@ -123,14 +123,9 @@ class BasicBoundingVolumeHierarchy
   node_type const *getRoot() const { return _internal_and_leaf_nodes.data(); }
 
   KOKKOS_FUNCTION
-  bounding_volume_type const &getRootBoundingVolume() const
+  bounding_volume_type const *getRootBoundingVolumePtr() const
   {
-    static_assert(
-        std::is_reference<decltype(Details::HappyTreeFriends::getBoundingVolume(
-            *this, Details::HappyTreeFriends::getRoot(*this)))>::value,
-        "");
-    return Details::HappyTreeFriends::getBoundingVolume(
-        *this, Details::HappyTreeFriends::getRoot(*this));
+    return &_internal_and_leaf_nodes.data()->bounding_volume;
```

Comment:
This is nasty.  We can't go through view accessor or we get a memory access violation.

Reply:
We need the address to the bounding box of the root to copy it back onto the host and we can't access elements of `_internal_and_leaf_nodes` from the BVH constructor since the data is on the device.

Reply:
want me to assert `getRoot()` returns zero here?